modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc

```diff
@@ -1,50 +1,45 @@
 
-= Creating Client Applications with Connected Apps
+= Creating Client Applications with the Connected App Feature
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: grant access, api, contracts
 
 
-For automation use cases, you can use Connected Apps to create and manage client applications.
+For automation use cases, you can use the Connected App feature to create and manage client applications.
 
-Connected Apps help users delegate their access without sharing sensitive credentials or giving full control of their applications to third parties. Actions taken by connected apps are audited, and users can revoke access at any time.
+Connected apps help users delegate their access without sharing sensitive credentials or giving full control of their applications to third parties. Actions taken by connected apps are audited, and users can revoke access at any time.
 
 
 == Creating a Connected App
 
-To create a Connected App, see xref:access-management::connected-apps-developers.adoc[].
+To create a connected app, see xref:access-management::connected-apps-developers.adoc[]
```

Comment:
I still think this needs terminating punctuation...

Reply:
So right!